ui/src/components/useMessageDialog.tsx

```diff
@@ -111,12 +117,23 @@ const MessageDialogContainer = () => {
     const node = document.createElement("div");
     document.body.appendChild(node);
     messageDialogRef.current = node;
+    register("message", {
+      required: "Please enter a message",
+      minLength: {
+        value: 3,
+        message: "Please enter a message with a minimum of 3 characters",
+      },
+    });
   }, []);
 
-  const postMessage = () => {
+  React.useEffect(() => {
+    if (textareaRef.current) textareaRef.current.value = "";
```

Comment:
i assume the check is for typescript, but in this case if textareaRef is set in the component would it make more sense for it to be never null?

Reply:
How do you mean? On initial declaration it will be null right? So the type has to cater for null.

Anway, as above, will be replacing this usage of ref with key on the textarea.